modules/porous_flow/doc/content/source/materials/PorousFlowMassFraction.md

```diff
@@ -2,6 +2,33 @@
 
 !syntax description /Materials/PorousFlowMassFraction
 
+The `mass_fraction_vars` is a list of mass fractions, $\chi_{\beta}^{\kappa}$, of fluid component $\kappa$ in phase $\beta$:
```

Comment:
This is much better for users. Thanks Andy

Reply:
Thanks for the review @cpgr .    The reason for this change is i got confused when helping Scott H while you were away.  I couldn't remember which mass fractions we had to specify!